st/clitests/jclient_spec.py

```diff
@@ -756,7 +756,20 @@ def get_response_elements(response):
 
     JClientTest('Jclient can delete bucket').delete_bucket("seagatebucket").execute_test().command_is_successful()
 
+# ************ TEST: Multiple INIT-MPU ***********
+    JClientTest('Jclient can create bucket seagatebucket').create_bucket("seagatebucket").execute_test().command_is_successful()
+
+    JClientTest('Jclient can initiate multipart upload').init_mpu("seagatebucket", "18MBfile", 18000000)\
+            .execute_test().command_is_successful()
+
+    JClientTest('Jclient can again initiate multipart upload').init_mpu("seagatebucket", "18MBfile", 18000000)\
+            .execute_test().command_is_successful()
 
+    result = JClientTest('Jclient can list all multipart uploads.').list_multipart("seagatebucket").execute_test()
+    result.command_response_should_have('18MBfile')
```

Comment:
command_response_should_have('18MBfile')  A new function you can define, ie command_response_should_have_n_times('18MBfile',2) that function sees whether 18MBfile occured 2 times

Reply:
done